{
    "assignee": null,
    "assignees": [],
    "author_association": "NONE",
    "body": "windows 10\r\nanaconda Version 5.3\r\nPython 3.7.0\r\nMSC v.1912 64 bit (AMD64)\r\n\r\nI had this function on jupyter lab.\r\n\r\n    def fib(n):\r\n        if n<2:\r\n            return n\r\n        return fib(n-1)+fib(n-2)\r\n\r\ni call it with code below and it ran well.\r\n\r\n    fib(20)\r\n\r\nThen i tried to use cython. First i load cython.\r\n\r\n    %load_ext Cython\r\n\r\nAnd here was where i used cython.\r\n\r\n    %%cython\r\n    def fib_cython(n):\r\n        if n<2:\r\n            return n\r\n        return fib_cython(n-1)+fib_cython(n-2)\r\n\r\nAfter i ran the code above,  this error cames up\r\n\r\n    DistutilsPlatformError: Unable to find vcvarsall.bat\r\n\r\nI've tried following instruction on this link\r\n\r\nhttps://www.ibm.com/developerworks/community/blogs/jfp/entry/Installing_Cython_On_Anaconda_On_Windows?lang=en#action=addcomment\r\n\r\nIt didn't work for me.\r\n\r\nIs there anyone know how to solve this problem?",
    "closed_at": null,
    "comment_data": [
        {
            "author_association": "NONE",
            "body": "I have the same issue. I followed the instructions for [Cython working on Windows](https://github.com/cython/cython/wiki/CythonExtensionsOnWindows) but didn't get past `setenv` isn't a Windows shell command. \r\n\r\n",
            "created_at": "2018-11-13T00:01:43Z",
            "html_url": "https://github.com/cython/cython/issues/2711#issuecomment-438076966",
            "id": 438076966,
            "issue_url": "https://api.github.com/repos/cython/cython/issues/2711",
            "node_id": "MDEyOklzc3VlQ29tbWVudDQzODA3Njk2Ng==",
            "updated_at": "2018-11-13T00:01:43Z",
            "url": "https://api.github.com/repos/cython/cython/issues/comments/438076966",
            "user": {
                "avatar_url": "https://avatars2.githubusercontent.com/u/12815335?v=4",
                "events_url": "https://api.github.com/users/josephcslater/events{/privacy}",
                "followers_url": "https://api.github.com/users/josephcslater/followers",
                "following_url": "https://api.github.com/users/josephcslater/following{/other_user}",
                "gists_url": "https://api.github.com/users/josephcslater/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/josephcslater",
                "id": 12815335,
                "login": "josephcslater",
                "node_id": "MDQ6VXNlcjEyODE1MzM1",
                "organizations_url": "https://api.github.com/users/josephcslater/orgs",
                "received_events_url": "https://api.github.com/users/josephcslater/received_events",
                "repos_url": "https://api.github.com/users/josephcslater/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/josephcslater/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/josephcslater/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/josephcslater"
            }
        }
    ],
    "comments": 1,
    "comments_url": "https://api.github.com/repos/cython/cython/issues/2711/comments",
    "created_at": "2018-11-12T23:33:25Z",
    "event_data": [],
    "events_url": "https://api.github.com/repos/cython/cython/issues/2711/events",
    "html_url": "https://github.com/cython/cython/issues/2711",
    "id": 379994990,
    "labels": [],
    "labels_url": "https://api.github.com/repos/cython/cython/issues/2711/labels{/name}",
    "locked": false,
    "milestone": null,
    "node_id": "MDU6SXNzdWUzNzk5OTQ5OTA=",
    "number": 2711,
    "repository_url": "https://api.github.com/repos/cython/cython",
    "state": "open",
    "title": "unable to use cython anaconda on windows 10",
    "updated_at": "2018-11-13T00:01:43Z",
    "url": "https://api.github.com/repos/cython/cython/issues/2711",
    "user": {
        "avatar_url": "https://avatars3.githubusercontent.com/u/44483328?v=4",
        "events_url": "https://api.github.com/users/tempegarit/events{/privacy}",
        "followers_url": "https://api.github.com/users/tempegarit/followers",
        "following_url": "https://api.github.com/users/tempegarit/following{/other_user}",
        "gists_url": "https://api.github.com/users/tempegarit/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/tempegarit",
        "id": 44483328,
        "login": "tempegarit",
        "node_id": "MDQ6VXNlcjQ0NDgzMzI4",
        "organizations_url": "https://api.github.com/users/tempegarit/orgs",
        "received_events_url": "https://api.github.com/users/tempegarit/received_events",
        "repos_url": "https://api.github.com/users/tempegarit/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/tempegarit/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/tempegarit/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/tempegarit"
    }
}